.github/actions/install-dashboards/action.yml

```diff
@@ -54,32 +62,75 @@ runs:
       run: git checkout ${{ steps.osd-version.outputs.osd-version }} || git checkout ${{ steps.osd-version.outputs.osd-x-version }}
       working-directory: ./OpenSearch-Dashboards
       shell: bash
+      
+    - name: Get and Setup OpenSearch-Dashboards Windows
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        curl -SLO https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/windows/x64/zip/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+        unzip -q opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+      shell: bash
 
+    - name: Get and Setup OpenSearch-Dashboards Linux
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/linux/x64/tar/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+        tar -xzf opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+      shell: bash
+    
     - id: tool-versions
       run: |
-        echo "node_version=$(cat .node-version)" >> $GITHUB_OUTPUT
+        echo "node_version=$(cat .node-version)" >> $GITHUB_ENV
         echo "yarn_version=$(jq -r '.engines.yarn' package.json)" >> $GITHUB_OUTPUT
       working-directory: OpenSearch-Dashboards
       shell: bash
 
-    - uses: actions/setup-node@v1
+    - name: Setup node
+      uses: actions/setup-node@v1
       with:
-        node-version: ${{ steps.tool-versions.outputs.node_version }}
+        node-version: ${{ env.node_version }}
         registry-url: 'https://registry.npmjs.org'
 
-    - name: Setup Opensearch Dashboards
-      run: |
-        npm uninstall -g yarn
-        echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
-        npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
-        yarn cache clean
-        yarn add sha.js
-      working-directory: OpenSearch-Dashboards
-      shell: bash
-
     - name: Bootstrap the OpenSearch Dashboard
       uses: nick-fields/retry@v2
       with:
         timeout_minutes: 20
         max_attempts: 2
         command: yarn --cwd OpenSearch-Dashboards osd bootstrap --oss
+
+    - id: build-plugin
+      run: |
+        yarn build
+      working-directory: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
+      shell: bash
+    
+    - name: Remove Security Plugin
```

Comment:
These removal steps don't seem right, we should be installing the dashboards standalone which wouldn't have any other plugins installed.  Lets update the installer and remove these 'removal' steps

If we have other plugins installed, this codebases CI stability is impacted by any plugin that introduces instability - ideally this repo should only be impacted by the stability of dashboards 🤣

Reply:
Agreed, I was trying to pull core min distribution, but due to the instability will be opting for checking out and building locally, updates to come soon.